core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -87,8 +88,8 @@ private[spark] class SparkHadoopUtil extends Logging {
   }
 
   /**
-   * Appends spark.hadoop.* configurations from a [[SparkConf]] to a Hadoop
-   * configuration without the spark.hadoop. prefix.
+   * Appends spark.hadoop.* configurations from a [[SparkConf]] to a Hadoop configuration without
```

Comment:
ditto

Reply:
Should I apply fmt only to the new code? That unclear

Reply:
Rolled back unnecessary formatting